### PR TITLE
Feature/make relationshipstab not use global submit button

### PR DESCRIPTION
**Pull request by wipascal**

### CODE REVIEW

File: basxconnect/core/layouts/relationshipstab.py [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    header: str,
+    field_name: str,
+    get_person: Callable[[Relationship], Person],
+) -> DataTableColumn:
+    return DataTableColumn(
+        header,
+        hg.SPAN(
+            person_name(field_name),
+            person_number_in_brackets(field_name),
+            **attributes_for_link_to_person(get_person),
```
Comment (saemideluxe) [inline basxconnect/core/layouts/relationshipstab.py:137]: This one you might want to assign to the DataTableColumn.td_attributes.
I think the SPAN will not fill the whole cell by default.

Comment (saemideluxe) [reply basxconnect/core/layouts/relationshipstab.py:137]: (I mean only taking the ```attributes_for_link_to_person(get_person)``` part out, the SPAN is not a problem)

**Review by saemideluxe — APPROVED** [review]
Okay, looks clean to me.
Looking at how we use ModelHref we should maybe add a bit of parameterization in the future for things like the ajax url and returning to the current page.

File: basxconnect/core/layouts/relationshipstab.py (hunk 1 above)
Comment (saemideluxe) [reply basxconnect/core/layouts/relationshipstab.py:137]: Whether you really want to change this depends of course on how you want the UI to behave. If the full-cell-click is not desired then there is no need to change it.

Comment (wipascal) [reply basxconnect/core/layouts/relationshipstab.py:137]: I just reverted this from an old version where we had these `attributes_for_link_to_person`... I think frederik wants to redesign it anyway.
Currently I am only trying to get rid of the global submit button :)

**Pull request merged**